dati_playground/validators/__init__.py

```diff
@@ -6,27 +7,17 @@
 import yaml
 from rdflib import Graph
 
-from .utils import MIME_JSONLD, MIME_TURTLE, yaml_to_json
+import dati_playground.validators.json_schema as json_schema
+import dati_playground.validators.openapi as openapi
+import dati_playground.validators.turtle as turtle
 
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger(__name__)
 
-def true(*a, **kw):
-    return True
-
-
-def is_jsonschema(content: str):
-    jsonschema.Draft7Validator.check_schema(yaml.safe_load(content))
-
-    return True
-
+from dati_playground.utils import MIME_JSONLD, yaml_to_json
 
-def is_openapi(content: str):
-    from openapi_spec_validator import validate_spec
-
-    spec_dict = yaml.safe_load(content)
-
-    # If no exception is raised by validate_spec(), the spec is valid.
-    validate_spec(spec_dict)
 
+def true(*a, **kw):
```

Comment:
Is this used somewhere?

Reply:
I think not. But I didn't want to eliminate what was there before, regardless of its usefulness. My job wasn't to overhaul the entire project, but to integrate new hooks.